### ISSUE TITLE
Center_of_mass returns int instead of float

### ISSUE THREAD

**Issue by aschueth** [dask/dask-image#120]
* dask-image version: 0.2.0
* Python version: 3.7.3
* Operating System: Linux

In scipy.ndimage.measurements.center_of_mass, the function returns indices of type float, whereas dask_image.ndmeasure.center_of_mass returns indices of type int. It seems like a relatively small difference, but to duplicate functionality it would be nice if this function could return floats.

**Comment by jakirkham** [comment]
Yep, that's seems reasonable. Thanks for raising, @aschueth.

**Comment by GenevieveBuckley** [comment]
Here is a reproducible example of the behaviour. 

Everything works as expected for arrays containing floats:
```python
import numpy as np
import dask.array as da
import dask_image.ndmeasure

arr = np.array([[1, 1],[0, 0]]).astype(float)
dask_arr = da.from_array(arr)
result = dask_image.ndmeasure.center_of_mass(dask_arr)
result.compute()
```
This gives us the result we expect: `array([0. , 0.5])`.

And we can reproduce the bug simply by passing in an array containing integer values instead. 
```python
arr = np.array([[1, 1],[0, 0]]).astype(int)
dask_arr = da.from_array(arr)
result = dask_image.ndmeasure.center_of_mass(dask_arr)
result.compute()
```

The output here is `array([0, 0])` instead of the expected `array([0. , 0.5])`.

**Comment by GenevieveBuckley** [comment]
And as mentioned, scipy can handle integer array input just fine:
```python
import numpy as np
import scikpy.ndimage as ndi
arr = np.array([[1, 1],[0, 0]]).astype(int)
result = ndi.center_of_mass(arr)
result
```
...which gives us the result we expect:
```
(0.0, 0.5)
```

**Comment by GenevieveBuckley** [comment]
Work started on this issue in https://github.com/dask/dask-image/pull/122

**Comment by GenevieveBuckley** [comment]
This is fixed in the master branch now that #122 is merged.
